e2e_tests/.flake8

```diff
@@ -36,3 +36,9 @@ import-order-style = edited
 # flake8-quotes
 inline-quotes = "
 multiline-quotes = """
+
```

Comment:
https://github.com/determined-ai/determined/pull/2140 should remove the need for this change but in the meantime we can do something like this